docs/INSTALLATION.md

```diff
@@ -1,13 +1,27 @@
 # Installation
 
-Install the essential build tools, the QEMU emulator, and the GCC ARM64 cross-toolchain to test your code.
+## Ubuntu
```

Comment:
The exact same packages will be available on Debian as well. Include that in the title maybe?